apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -219,9 +235,9 @@ export default function BaseMegaMenu() {
                         </ul>
                       </div>
                     ))}
-                    <div className="flex flex-col items-center justify-center bg-neutral-100 rounded-md border-neutral-300 overflow-hidden grow">
+                    <div className="flex flex-col items-center justify-center overflow-hidden rounded-md bg-neutral-100 border-neutral-300 grow">
```

Comment:
Banner is rounded on mobile as well, but it not supposed to be.
<img width="322" alt="Screenshot 2023-04-21 at 12 09 47" src="https://user-images.githubusercontent.com/34419118/233611753-0a82e1fc-18c9-409f-9232-989cdb473f59.png">